src/main/java/io/jenkins/plugins/pipelinegraphview/utils/PipelineStepApi.java

```diff
@@ -45,13 +45,17 @@ private List<PipelineStep> parseSteps(List<FlowNodeWrapper> stepNodes, String st
                     displayName = cleanTextContent(displayName);
                     logger.debug("DisplayName After: '" + displayName + "'.");
 
+                    // Ignore certain titles
+                    if (title.equals("Shell Script") || title.equals("Print Message")) {
```

Comment:
May need to check display name is set. I think there’s some times when the display name is thrown away with shell scripts

Reply:
Wrapped it in an `if` to confirm display name is set.